src/CSET/operators/plot.py

```diff
@@ -59,16 +122,15 @@ def _check_single_cube(
 
 
 def spatial_contour_plot(
-    cube: iris.cube.Cube, file_path: Path, **kwargs
+    cube: iris.cube.Cube, filename: Path = None, **kwargs
 ) -> iris.cube.Cube:
-    """
-    Plot a spatial variable onto a map.
+    """Plot a spatial variable onto a map.
 
     Parameters
     ----------
     cube: Cube
         An iris cube of the data to plot. It should be 2 dimensional (lat and lon).
-    file_path: pathlike
+    filename: pathlike
```

Comment:
What do you mean exactly? Put a directory name or a filename?

Reply:
A [pathlike](https://docs.python.org/3/library/os.html?highlight=pathlike#os.PathLike) is something that can act as a file path. So either a string or a pathlib.Path().

For the filename parameter it should just be the name of the plot file desired, though it is optional. I'll update the docstring to make it clearer.